src/documents/ComponentConfigurationOptions.md

```diff
@@ -179,80 +195,51 @@ The following options are supported by all component grades:
-  </tr>
-  <tr>
-    <th>Notes</th>
-    <td>Some special context names may be available within the subcomponent's definition block, for example <code>{source}</code> and <code>{sourcePath}</code> or <code>{arguments}</code>. This framework facility will be replaced by a more declarative equivalent in time and should be used with caution.</td>
+    <td>A record directing the framework to distribute options from this component to one or more other components in the component tree. Either a single, an <code>Array</code> or <code>Object</code> holding these records is supported.
+    In the <code>Object</code> form, the keys of the object will be taken to represent the <code>namespace</code of the distribution.</td>
   </tr>
   <tr>
     <th>Example Definition</th>
     <td><pre>
 <code>fluid.defaults("component.name", {
-    dynamicComponents: {
-        dynamic1: {
-            type: "component.subcomp1",
-            source: "{context}.someArray",
-            options: {...}
-        },
-        ...
-    },
+    gradeNames: ["fluid.component"],
+    distributeOptions: {
+        namespace: "myDistribution",
+        record: "another.grade.name",
+        target: "{that target.grade.name}.options.gradeNames"
+    }
     ...
 });</code>
 </pre></td>
   </tr>
   <tr>
     <th>See also</th>
-    <td><a href="tutorial-gettingStartedWithInfusion/Subcomponents.md">Tutorial: Subcomponents</a></td>
+    <td><a href="IoCSS.md">IoCSS for options distributions</a></td>
   </tr>
 </table>
 
-
-## Little Components ##
-
-Components defined with a grade of `littleComponent` support all of the [common options](#options-supported-by-all-components-grades) described above, and no others. Component developers are free to define their own additional options.
-
-See also: [Component Grades](ComponentGrades.md)
-
-## Model Components ##
-
-Components defined with a grade of `modelComponent/modelRelayComponent` support all of the [common options](#options-supported-by-all-components-grades) described above, as well as those defined below. Component developers are free to define their own additional options.
-
-See also: [Component Grades](ComponentGrades.md)
-
-### model ###
+### `mergePolicy` ###
 <table>
   <tr>
     <th>Description</th>
-    <td>An object containing the data model to be used by the component.</td>
+    <td>An object providing instructions for how particular options should be merged when integrator options are merged with default values.</td>
   </tr>
   <tr>
     <th>Notes</th>
-    <td>If a <a href="ChangeApplier.md">ChangeApplier</a> is not provided using the <code><a href="#applier">applier</a></code> option, the Framework will create one for the provided model. </td>
+    <td>It is uncommon to need this option. The most common use case is to protect "exotic values" derived from some external library or framework from being corrupted by the options merging/expansion process by use of the "nomerge" policy.</td>
```

Comment:
Is there an example of a kind of "exotic value" that one should typically protect? Or conversely, is there an example of a value that might seem like it needs be guarded, but doesn't? What about, say, a reference to a DOM object or other browser-implemented object (e.g. a Blob or an AudioContext or an Element)?